apps/showcase/src/app/home/home.template.html

```diff
@@ -19,20 +19,40 @@ <h3 class="display-3 text-center mt-0 mb-5">Get started with Otter</h3>
         <h3>Setup the project ...</h3>
         <p class="mt-4 text-secondary">
           Create a new project <i>(npm)</i>
-          <o3r-copy-text-pres language="bash" text="npm create @o3r <your-project>"></o3r-copy-text-pres>
+          <markdown clipboard ngPreserveWhitespaces>
+            ```bash
+            npm create &#64;o3r &lt;your-project&gt;
+            ```
+          </markdown>
           or <i>(yarn)</i>
-          <o3r-copy-text-pres language="bash" text="yarn create @o3r <your-project>"></o3r-copy-text-pres>
+          <markdown clipboard ngPreserveWhitespaces>
+            ```bash
+            yarn create &#64;o3r &lt;your-project&gt;
```

Comment:
Sorry same remark as previously:

```suggestion
            npm create &#64;o3r &lt;your-project&gt; --yarn
```